tests/test_model.py

```diff
@@ -612,23 +646,42 @@ def test_forward_with_padding(attention_impl: str, device: str, alibi: bool):
                              attention_mask=batched_attention_mask).logits
 
         # check that right padding and left padding produce the same output
-        assert torch.allclose(right_padding_output[0, :3],
-                              left_padding_output[0, 3:],
-                              atol=1e-6 if attention_impl == 'torch' else 1e-8)
-        if not alibi:
+        if rope and pos_emb_config[
```

Comment:
lets just define the expected tolerance in the if statement instead of the full assert

Reply:
Done.